Tools/Eflow/install-hyperv.ps1

```diff
@@ -0,0 +1,15 @@
+<# .SYNOPSIS
+    Hyper-V installation Script
+.DESCRIPTION
+    This script automatically installs Hyper-V on a Windows Server VM to allow eflow provisioning.
+    The VM requires a restart after this script is run before eflow can be provisioned.
+.NOTES
+     Author     :
+        Mikhail Chatillon       - chmikhai@microsoft.com
+        Daniele Antonio Maggio  - daniele.maggio@microsoft.com
+#>
+
+# Install Hyper-V
+Install-WindowsFeature -Name Hyper-V
+Install-WindowsFeature -Name Hyper-V-PowerShell
+Enable-WindowsOptionalFeature -Online -FeatureName VirtualMachinePlatform -All -NoRestart
```

Comment:
We should either restart the machine or ask the user to restart it here

Reply:
It's a step in the readme

Reply:
Yep, can we also print a log saying "Please reboot"? from the script? 👍

Reply:
The script prints out a log that indicates that a restart is needed. But I will add an extra message :)